errors/public.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 Security Scorecard Authors
+//
```

Comment:
ignore this file, it will disappear once previous https://github.com/ossf/scorecard/pull/712 and https://github.com/ossf/scorecard/pull/718 are merged